tools/PI/DevHome.PI/BarWindowHorizontal.xaml.cs

```diff
@@ -0,0 +1,337 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+using DevHome.PI.Controls;
+using DevHome.PI.Helpers;
+using DevHome.PI.Models;
+using DevHome.PI.Properties;
+using DevHome.PI.ViewModels;
+using Microsoft.UI.Input;
+using Microsoft.UI.Windowing;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+using Windows.Foundation;
+using Windows.UI.WindowManagement;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+using Windows.Win32.UI.WindowsAndMessaging;
+using WinRT.Interop;
+using WinUIEx;
+using static DevHome.PI.Helpers.WindowHelper;
+
+namespace DevHome.PI;
+
+public partial class BarWindowHorizontal : WindowEx
+{
+    private readonly Settings settings = Settings.Default;
```

Comment:
Dev Home uses _ prefixes for private class variables.

Reply:
I've updated that for all the new files I've added and the files I've made major changes to. We'll continue to update as we make more changes.